geocode_array/Geocoder.py

```diff
@@ -8,8 +8,11 @@
 from geocode_array import REQUEST_HEADER_DICT, REQUEST_TRIES, REQUEST_DELAY
 
 
-def _form_request(request_base_url, request_url_args, proxy_url) -> urllib.request.Request:
-    request_string = f"{request_base_url}?{request_url_args}"
+def _form_request(request_base_url, request_url_args, proxy_url, bing_reverse=False) -> urllib.request.Request:
```

Comment:
/nitpick

The difference is between path vs query string parameters, as opposed to anything M$ crazily came up with - suggest the flag is rather called `path_parameters`.

Reply:
added done